glusterd2/brick/types.go

```diff
@@ -24,10 +24,10 @@ const (
 
 //MountInfo is used to store mount related information of a volume
 type MountInfo struct {
-	Mountdir   string
-	DevicePath string
-	FsType     string
-	MntOpts    string
+	BrickDirSuffix string
```

Comment:
What does `BrickDirSuffix` mean ? What's wrong with `Mountdir` ?

The `MountInfo` structure is analogous to the output of mount command.

Reply:
The mountdir used to store directory name after mount root. for example if the directory is mounted as /bricks/gluster and the brick path is /bricks/gluster/b1 then we store /b1 as mountdir . This name was causing confusion with mounted directory ie /bricks/gluster